keps/sig-auth/3257-trust-anchor-sets/README.md

```diff
@@ -264,23 +264,26 @@ connections to the server without downtime, even as I rotate the trust anchors
 that back the CA.
 
 To do this, I update the controller that implements `example.com/server-tls` to
-create a TrustAnchorSets object like this one:
+create a ClusterTrustBundle object like this one:
 ```yaml
 apiVersion: v1alpha1
-kind: TrustAnchorSets
+kind: ClusterTrustBundle
```

Comment:
Is it realistic that there could in the future be a namespaced version of this resource?
(Mostly asking because I am interested in using this mechanism in cert-manager, but would like to avoid giving our namespaced issuers ability to create/update cluster-scoped resources)

Reply:
Other reviewers requested this as well.  The ClusterTrustBundle naming is meant to explicitly leave this possibility open.

Reply:
Namespaced TrustBundle _could_ be a CRD, albeit an official Kubernetes one.

Reply:
> Ah no, the kubelet needs to know about it.

So the need to have kubelet watch another type would likely be a reason to not add this or just that this would require more work than just adding another type?

Reply:
I think we'll have our work cut out for us to land the existing scope in 1.26, but I definitely want to leave the door open for this.